Conversions/Lengthconversions.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * Class for converting length between different units.
+ */
+class LengthConversions {
+    /**
+     * Validates input for conversion methods
+     * 
+     * @param mixed $value The value to check
+     * @param string $method The method name for error message
+     * @throws InvalidArgumentException
+     */
+    private static function validateInput($value, $method) {
+        // Make sure we have a numeric value
+        if (!is_numeric($value)) {
+            throw new InvalidArgumentException("Invalid input for $method");
```

Comment:
throw new InvalidArgumentException("Invalid input for $method: expected numeric, got " . gettype($value) . " (" . var_export($value, true) . ")");